dali/base/dadfs.cpp

```diff
@@ -9131,6 +9158,9 @@ class CInitGroups
 
     bool loadMachineMap()
     {
+        if (machinesLoaded)
+            return true;
+        machinesLoaded = true;
```

Comment:
maybe would look more logical if was at end of method.
Shouldn't make any difference as not protected/not expected to be thread-safe